gapis/trace/tracer/default_api_trace_options.go

```diff
@@ -22,7 +22,7 @@ import (
 func VulkanTraceOptions() *service.TraceTypeCapabilities {
 	return &service.TraceTypeCapabilities{
 		Type:                           service.TraceType_Graphics,
-		Api:                            "Vulkan",
+		Api:                            "Vulkan (experimental)",
```

Comment:
Does this work? I would expect this to break things in fun ways when the UI tries to take a trace and passes this string back to the server.

Reply:
works now, miss a string compare

Reply:
As the failure in the build shows, this is too far-reaching of a change. Instead of changing the API here, just change the label in the UI. In the `createApiDropDown(..)` function, simple change the Graphics case to: `return ttc.getApi() + " (Experimental)"`.